views/layout.hbs

```diff
@@ -61,9 +36,7 @@
 
   <body class="{{ bodyClasses }}">
     <script type="text/javascript">document.body.className = ((document.body.className) ? document.body.className + ' js-enabled' : 'js-enabled');</script>
-
-    {{{ bodyStart }}}
-
+        {{{ bodyStart }}}
```

Comment:
inconsistent indentation, either use 4 or 2 spaces and stick to it over the whole doc